_episodes/07-vector-data-in-python.md

```diff
@@ -57,42 +72,49 @@ bbox = (xmin, ymin, xmax, ymax)
 ~~~
 {: .language-python}
 
-> ## How should I define my bounding box?
-> For simplicity, here we assume the **Coordinate Reference System (CRS)** and **extent** of the vector file are known (for instance they are provided in the dataset documentation). Some Python tools, e.g. [`fiona`](https://fiona.readthedocs.io/en/latest/)(which is also the backend of `geopandas`), provides the file inspection functionality without actually the need to read the full data set into memory. An example can be found in [the documentation of fiona](https://fiona.readthedocs.io/en/latest/manual.html#format-drivers-crs-bounds-and-schema).
-{: .callout}
-
 Using the `bbox` input argument, we can load only the spatial features intersecting the provided bounding box.
 
 ~~~
 # Partially load data within the bounding box
-cropfield = gpd.read_file("data/brpgewaspercelen_definitief_2020_small.gpkg", bbox=bbox)
+fields = gpd.read_file("data/brpgewaspercelen_definitief_2020_small.gpkg", bbox=bbox)
 ~~~
 {: .language-python}
 
-## Vector Metadata & Attributes
-When we import the vector dataset to Python (as our `cropfield` object) it comes in as a `DataFrame`, specifically a `GeoDataFrame`. The `read_file()` function also automatically stores
-geospatial information about the data. We are particularly interested in describing the format, CRS, extent, and other components of
-the vector data, and the attributes which describe properties associated
-with each individual vector object.
+> ## How should I define my bounding box?
+> For simplicity, here we assume the **Coordinate Reference System (CRS)** and **extent** of the vector file are known (for instance they are provided in the dataset documentation). 
+> 
+> You can also define your bounding box with online coordinates visualization tools. For example, in our case, we can use the [RD-viewer](https://openstate.github.io/rdnewviewer/).
```

Comment:
reading the changes further, I see why you selected this particular coordinate viewer, it looks like it is specific to this Dutch projection. I think it would still be useful to show a coordinate viewer like geojson.io since users will usually be needing to draw AOIs with lat lon coordinates rather than use RD Viewer. Can we mention both at make the distinction that most coordinate viewers use EPSG 4326 and lat/lon, and that RD Viewer in particular uses the RD projection?

Reply:
Thanks Ryan! I think it's a good idea. Living in the Netherlands does make me forget the international standards a bit. Will change this

Reply:
@rbavery I used your suggestion, kept the geojson.io with WGS84 as a default, and mentioning since we are working with a Dutch dataset, we can use the RD viewer.